raft/src/test/java/org/apache/kafka/raft/LeaderStateTest.java

```diff
@@ -183,164 +181,6 @@ public void testNonMonotonicLocalEndOffsetUpdate() {
         );
     }
 
-    @ParameterizedTest
```

Comment:
Did you consider replacing these tests instead of completely removing them? Why did you decide to remove them?

Reply:
They all are covered by new test cases I added in KafkaRaftClientTest